tomviz/pvextensions/CMakeLists.txt

```diff
@@ -20,8 +20,17 @@ add_paraview_plugin(tomvizExtensions "1.0"
 )
 target_link_libraries(tomvizExtensions PUBLIC pqApplicationComponents)
 
-install(TARGETS tomvizExtensions
-        RUNTIME DESTINATION "bin"
-        LIBRARY DESTINATION "lib"
-        ARCHIVE DESTINATION "lib"
-	COMPONENT "runtime")
+if (NOT APPLE)
+  install(TARGETS tomvizExtensions
+          RUNTIME DESTINATION "bin"
+          LIBRARY DESTINATION "lib"
+          ARCHIVE DESTINATION "lib"
+          COMPONENT "runtime")
+else()
+  install(TARGETS tomvizExtensions
+          RUNTIME DESTINATION "bin"
+          LIBRARY DESTINATION "Applications/tomviz.app/Contents/Libraries"
```

Comment:
You can just use BUNDLE DESTINATION in the main call, rather than using if/else I think, see 'cmake help-command install'. Do you know why this changed when bumping the minimum CMake version? I am guessing that is what happened. I am also pretty sure the BUNDLE location means it can be a relative path within the bundle, keeping this a little simpler.